reVX/hybrid_stats/hybrid_stats.py

```diff
@@ -0,0 +1,829 @@
+# -*- coding: utf-8 -*-
+"""
+Module to compute hybrid solar-wind generation correlations
+"""
+from concurrent.futures import as_completed
+import logging
+import numpy as np
+import os
+import pandas as pd
+from scipy.spatial import cKDTree
+from scipy.stats import pearsonr, spearmanr, kendalltau
+
+from rex.resource import Resource
+from rex.utilities.execution import SpawnProcessPool
+from rex.utilities.utilities import get_lat_lon_cols
+
+logger = logging.getLogger(__name__)
+
+
+class HybridStats:
+    """
+    Compute comparitive correlations for co-located wind and solar generation
+    """
+    def __init__(self, solar_h5, wind_h5, statistics='pearson',
+                 res_cls=Resource, hsds=False):
+        """
+        Parameters
+        ----------
+        solar_h5 : str
+            Path to solar h5 file(s)
+        wind_h5 : str
+            Path to wind h5 file(s)
+        statistics : str | tuple | dict, optional
+            Statistics to extract, either a key or tuple of keys in
+            cls.STATS, or a dictionary of the form
```

Comment:
Where is cls.STATS? I dont see it defined in this class?

Reply:
See below, its a hold over, based on how we decide to handle the default STATS I'll fix the docstring